users/app.py

```diff
@@ -60,5 +54,43 @@ def add_update_user():
     return jsonify(user_object), (201 if added_new_user else 200)
 
 
+def find_authorization_in_db(username, users_collection):
+    """Queries the db to find authorization of the given user.
+
+    Documents in the users collection should look like
+        {"username": "cmei4444",
+        "name": "Carolyn Mei",
+        "is_organizer": True}
+    """
+    if users_collection.count_documents({"username": username}) == 0:
+        return False  # user not found
+    cursor = users_collection.find({"username": username})
```

Comment:
Should this use [`find_one`](https://api.mongodb.com/python/current/api/pymongo/collection.html#pymongo.collection.Collection.find_one)?

Reply:
Ah, thank you. Fixed https://github.com/knative-portability/large-events/pull/2/commits/00a50b4cf0f70b251b63ffe424875b6979401b2e